controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -167,6 +170,8 @@ module.exports = async (req, res) => {
             ...userLocation,
             viewtype: 'user'
           });
+
+          allUserIds.push(userLocation.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure that the `user_id` is not already present in the `allUserIds` array before pushing it.
```diff
+           if (!allUserIds.includes(userLocation.user_id)) {
               allUserIds.push(userLocation.user_id);
+           }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->